.github/ISSUE_TEMPLATE/pipelines-scrum-template.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Pipelines Scrum Template
+about: 'Template for providing supplemental information on scrum cards. '
+title: ''
+labels: pipelines-scrum
+assignees: ''
+
+---
+
+## OUTCOME
+**What is the "done" condition of this card?**
+A clear and concise description of what the problem is.
+
+**What is the motivation for this card?**
```

Comment:
This is the feature, process, or task that is being capture by this issue.  I would put this first (issue or bug can be linked here).